vision/README.md

```diff
@@ -65,7 +65,7 @@ The vision team is responsible for processing images in order to extract informa
             ...
```

Comment:
This directory structure is dated-- it's missing folder such as module, text, pylon, etc.

It's not a huge deal, since users could just scroll up for a more accurate picture of the directory itself, but it could be useful for new members trying to figure out where to put/find stuff.

Reply:
My goal with that section was to provide a rough layout of the directory structure like what sorts of files should be in each folder.  Otherwise wed have to update it every time we add or remove a folder.